### ISSUE TITLE
Ocean registry minor ambiguity

### ISSUE THREAD

**Issue by alicebarthel** [E3SM-Project/E3SM#6204]
https://github.com/E3SM-Project/E3SM/blob/4cc2cedc0f0edb2ee0eb222ad5c50035e5cad16c/components/mpas-ocean/src/Registry.xml#L458-L460
The description of config_eddyMLD_use_old should state "keep old version instead of switching to ..." to be consistent with true vs. false of the "use_old" flag.

**Comment by xylar** [comment]
@alicebarthel, I agree. I think you should go ahead and make a PR to fix this (unless you're collecting multiple typos to make a combined PR for).
